main.py

```diff
@@ -2,9 +2,38 @@
 
 from db.models import Race, Skill, Player, Guild
 
+import json
+
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        players_info = json.load(f)
+    for pl in players_info.keys():
+        race, created = Race.objects.get_or_create(
+            name=players_info[pl]["race"]["name"],
+            description=players_info[pl]["race"]["description"]
+        )
+        if created and players_info[pl]["race"]["skills"]:
+            for skill in players_info[pl]["race"]["skills"]:
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=race
+                )
+        guild = None
+        if players_info[pl]["guild"]:
```

Comment:
Check if `players_info[pl]["guild"]` is not `None` before accessing its attributes to prevent potential `KeyError` or `TypeError`.